From 74782fe227819eeadc9768bd9ffb10ee81ef60e1 Mon Sep 17 00:00:00 2001
From: Anthony Iliopoulos <ailiop@altatus.com>
Date: Sat, 26 Oct 2019 12:45:46 +0200
Subject: [PATCH] liburing/examples: disable compilation of ucontext-cp example

the code leverages ucontext.h functionality which is not implemented in
musl libc and thus fails compilation. the example code is anyway not
part of the final package, so just remove it from compilation.
---
 examples/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git examples/Makefile examples/Makefile
index 6fe3ac3..61a2b56 100644
--- examples/Makefile
+++ examples/Makefile
@@ -4,7 +4,7 @@ override CFLAGS += -Wall -D_GNU_SOURCE -L../src/ -I../src/include/
 
 include ../Makefile.quiet
 
-all_targets += io_uring-test io_uring-cp link-cp ucontext-cp
+all_targets += io_uring-test io_uring-cp link-cp
 
 all: $(all_targets)
 
-- 
2.23.0

